.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,20 @@
+name: Build and Test
```

Comment:
I switched to Github actions because the travis check stopped running. I think because travis-ci.org is now shut down. I prefer Github actions anyway since it's faster.